Demo/src/main/java/com/braintreepayments/demo/MainActivity.java

```diff
@@ -152,7 +187,8 @@ public void launchDropIn(View v) {
             dropInRequest.setThreeDSecureRequest(demoThreeDSecureRequest());
         }
 
-        dropInClient.launchDropIn(dropInRequest);
+        // TODO: make auth string a required DropInRequest constructor parameter
+        dropInLauncher.launchDropIn(authString, dropInRequest);
```

Comment:
This reads a little odd to me since it's like the inverse of one another, not sure if something like this would make it feel less awward:
```suggestion
        dropInLauncher.start(authString, dropInRequest);
```

Reply:
Yeah def that's a good call. It aligns nicely with our SDKs as a whole. I'll do this in a separate commit to make sure all occurrences are renamed.